app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: ("Distance", int, float)) -> "Distance":
```

Comment:
Change your type annotation here and in the other places to use `|` and class instances without `""`
```suggestion
    def __add__(self, other: Distance | int | float) -> Distance:
```
Use `from __future__ import annotations` import in case you face errors

Reply:
Thank you for the suggestion, because I did it in the beginning but flake8 gave me the errors. but with __future__ everything became correct.